libminifi/test/unit/SocketTests.cpp

```diff
@@ -221,7 +222,7 @@ TEST_CASE("TestTLSContextCreation", "[TestSocket8]") {
  */
 TEST_CASE("TestTLSContextCreation2", "[TestSocket9]") {
   std::shared_ptr<minifi::Configure> configure = std::make_shared<minifi::Configure>();
-  configure->set("nifi.remote.input.secure", "false");
+  configure->set(org::apache::nifi::minifi::Configuration::nifi_remote_input_secure, "false");
```

Comment:
There is already a minifi namespace alias, use it!

Reply:
Updated in 107193764914b719074c45b1ce4b978c3d1292f4